turbinia/config/turbinia_config.py

```diff
@@ -45,6 +45,12 @@
 # NFS or a SAN for Evidence objects.
 SHARED_FILESYSTEM = False
 
+# This will set debugging flags for processes executed by Tasks (for
+# Tasks/binaries that support it).  This could cause performance issues with
```

Comment:
- extra space

Reply:
Where is the extra space?  I'm going to submit this for now, but happy to address this in a followup PR when I revamp the config.